glean-core/tests/common/mod.rs

```diff
@@ -7,6 +7,11 @@
 #![allow(dead_code)]
 use glean_core::Glean;
 
+use chrono;
```

Comment:
Yeah -- I liked having the fully-qualified names in this file below since there is a function that uses both `chrono::DateTime` and `iso8601::DateTime`.

Reply:
But crates are always in scope, so `chrono::DateTime` will work without that `use chrono` line.

Reply:
Oh, cool.  TIL.